cime_config/testdefs/testmods_dirs/clm/Fates/shell_commands

```diff
@@ -1,2 +1,3 @@
 ./xmlchange CLM_BLDNML_OPTS="-no-megan" --append
 ./xmlchange BFBFLAG="TRUE"
+./xmlchange CLM_FORCE_COLDSTART="on"
```

Comment:
We might want to fix this so that the cold start is only in the FatesCold testmod and everything that builds on it. This is putting it in the base class, so every FATES test is effectively a coldstart. It might be better to have the FatesCold in the testmod names and have it separate. That said I don't know that we have the capability to do FATES without a cold start? Can we setup some simple tower site tests that start from an inventory file.

Reply:
Per our discussion, I'll take care of this with a future PR addressing https://github.com/ESCOMP/CTSM/issues/1265#issuecomment-770113962